crypto-auditing/src/bpf/audit.h

```diff
@@ -12,7 +12,7 @@ typedef enum
     AUDIT_DATA_BLOB = 2,
   } audit_data_type_t;
 
-#define KEY_SIZE 32
+#define KEY_SIZE 64
```

Comment:
How is this change related to EMS event?

Reply:
The reason behind this is that the key name (`tls::ext::extended_master_secret`) doesn't fit in 32 bytes. Maybe we could use more generic way to represent extension advertisement / acceptance, something like:
`tls::ext::advertised` with extension ID as the value. Let me rewrite in that way.

Reply:
In the end I gave up on this idea, as the semantics of extension negotiation is defined per extension. So I simply bumped the key length to 64.